name_that_hash/hashes.py

```diff
@@ -1641,4 +1641,59 @@ class HashInfo:
                 )
         ],
     ),
+    Prototype(
+    regex=re.compile(r"^(bc1|[13])[a-zA-HJ-NP-Z0-9]{25,39}$", re.IGNORECASE),
+    modes=[
+        HashInfo(
+            name="Bitcoin / Litecoin",
+            hashcat=11300,
+            john="bitcoin",
+            extended=False,
+            description="Use Bitcoin2John.py to extract the hash for cracking."
+            )
+        ],
+    ),
+    Prototype(
+    regex=re.compile(r"^0x[a-fA-F0-9]{40}$", re.IGNORECASE),
```

Comment:
This regex also isn't right. An example Ethereum hash looks like this: `$ethereum$p*262144*3238383137313130353438343737383736323437353437383831373034343735*06eae7ee0a4b9e8abc02c9990e3730827396e8531558ed15bb733faf12a44ce1*e6d5891d4f199d31ec434fe25d9ecc2530716bc3b36d5bdbc1fab7685dda3946`

Reply:
This is solved too:

```
\$ethereum\$[a-z0-9*]{150,250}
```